components/collapsible-panel/collapsible-panel.js

```diff
@@ -171,6 +185,9 @@ class CollapsiblePanel extends FocusMixin(RtlMixin(LitElement)) {
 				display: flex;
 				margin-inline-start: var(--d2l-collapsible-panel-spacing-inline);
```

Comment:
Any chance you can sneak in this extra little layout fix since you're working on the layout? Currently the text is allowed to flow right to the edge.

I think it might just be a matter of adding `margin-inline-end: var(--d2l-collapsible-panel-spacing-inline);` for this `.d2l-collapsible-panel-header-secondary` element. It would end up being `0.9rem`/`18px;`. Then it will match up wit the content.

![image](https://user-images.githubusercontent.com/9042472/236330848-ef4707be-9b37-4df5-bf7d-fc7455f25080.png)